tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithRadiobox.spec.ts

```diff
@@ -84,7 +84,7 @@ test('can create content with the custom radiobox data type', async ({umbracoApi
   expect(contentData.values[0].value).toEqual(optionValues[0]);
 });
 
-test('can not publish mandatory radiobox with an empty value', async ({umbracoApi, umbracoUi}) => {
+test('can not publish mandatory radiobox with an empty value', {tag: '@release'}, async ({umbracoApi, umbracoUi}) => {
```

Comment:
Might not be needed, as we also have a test for 'can not publish a mandatory checkbox list with an empty value' with the release tag